uktrade/data-hub-frontend
# Cls2 541 viewing my tasks 2

## Description of change

Build a My Tasks table within the My tasks tab on the dashboard to display all tasks associated with an adviser. This list should display the due date in ascending order

## Test instructions

Run this branch pointing to Dev. The dashboard should display the My tasks tab and within that a table of all tasks associated to the logged in user. This will be both assigned to tasks and task that have been created by.

## Screenshots

![Screenshot 2023-11-29 at 16 56 26](https://github.com/uktrade/data-hub-frontend/assets/72826129/9e2c3022-ed50-4cf9-b830-7034e890885c)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'

- [x] Has the branch been rebased to feature branch?
